aem/aem_healthcheck_servlet.md

```diff
@@ -0,0 +1,49 @@
+   
+  
+## Trigger Adobe I/O Events health checks
+
+Once all your configurations are completed, in order to verify **and enable** 
+Adobe I/O Events emission, you must trigger the `Adobe I/O Events Servlet Health Check`.
+This servlet is available at `/bin/ioevents/checks` 
+
+   ![Health check Servlet output sample](../img/events_aem_healthcheck_servlet.png "Health check for conf-events")
+
+It exposes the following sections:
+
+###  AEM Adobe I/O Events Configuration checks
+
+This section shows your AEM identification configurations: 
+* the Link Externalizer name: by default set to `author`
+* the Externalizer author Root Url
+* the Adobe I/O Event Provider instance_id: derived from the above
+* the Adobe I/O Event Provider metadata: by default set to `aem`
+
+### Adobe IO Workspace Configuration checks
```

Comment:
Let's change `IO` to `I/O` throughout consistently.
```suggestion
### Adobe I/O Workspace Configuration checks
```

Reply:
sure